API/DataModel/Entities/OrderToServicePrice.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.DataModel.Entities.AspNetIdentity
+{
+    public class OrderToServicePrice
+    {
+        [Key]
```

Comment:
jw. ja bym zrobił wiele-do-wielu z poziomu fluent api, a tutaj dał navigation propertiesy. Będzie się prościej dostać w kontrolerach z zamówienia do usługi i odwrotnie.

Reply:
Kwestia gustu. Nie dałem w tej encji nawigacji, ale jakby była to coś w 
var order = repo.GetAll(null,null,x=>x.Include(x=>x.OrderToServicePrice).ThenInclude(x=>x.ServicePrice));

np. jakby trzeba było zwrócić ordersy ze wszystkimi dowiązeniami.